### PR TITLE
Support Demos

### PR DESCRIPTION
**Pull request by xinhaoc**
fix missing&misaligned part for running our demos

### CODE REVIEW

**Review by jiazhihao — APPROVED** [review]

**Comment by xinhaoc** [comment]
@jiazhihao do we have any specific lora&gqa_spec_decoding examples we want to test

**Comment by jiazhihao** [comment]
`demo/demo_gqa.py` is the gqa_spec_decoding example we want to test.

**Comment by jiazhihao** [comment]
We can merge this PR first if it is ready --- the GQA and LoRA test can be in a separate PR.

**Comment by xinhaoc** [comment]
> We can merge this PR first if it is ready --- the GQA and LoRA test can be in a separate PR.

it's ready now, for demo_lora and demo_group_query_attention_spec_decode, I can also add the generated graphs which we want to select

File: include/mirage/transpiler/runtime/kernel/element_unary.h [hunk 1]
```diff
@@ -9,19 +9,24 @@
...
+  }
   if constexpr (OP == ElementUnaryOpType::EXP) {
-    if constexpr (std::is_same_v<T, cutlass::half_t> ||
-                  std::is_same_v<T, __half>) {
-      return (T)expf((float)a);
-    } else {
-      assert(0);
-    }
+    return (T)expf((float)a);
+  } else if (OP == ElementUnaryOpType::SILU) {
```
Comment (jiazhihao) [inline include/mirage/transpiler/runtime/kernel/element_unary.h]: Checking operator type at run time is very expensive. Can we add `constexpr` as a quantifier for all the following if conditions?

File: include/mirage/transpiler/runtime/threadblock/element_unary.h [hunk 2]
```diff
@@ -11,20 +11,30 @@ using namespace cute;
...
-    if constexpr (std::is_same_v<T, cutlass::half_t> ||
-                  std::is_same_v<T, __half>) {
-      return (T)expf((float)a);
-    } else {
-      assert(0);
-    }
+    return (T)expf((float)a);
+  } else if constexpr (OP == ElementUnaryOpType::SILU) {
+    return (T)(((float)a) * (1.0f / (1.0f + expf((float)a))));
+  } else if (OP == ElementUnaryOpType::SQUARE) {
```
Comment (jiazhihao) [inline include/mirage/transpiler/runtime/threadblock/element_unary.h]: Similarly, let's add `constexpr` as a quantifier for all if conditions below.

**Review by jiazhihao — CHANGES_REQUESTED** [review]

File: include/mirage/transpiler/runtime/kernel/element_unary.h (hunk 1 above)
Comment (xinhaoc) [reply include/mirage/transpiler/runtime/kernel/element_unary.h]: done!

**Review by jiazhihao — APPROVED** [review]

**Pull request merged**